app/payment/pre-payment.controller.test.js

```diff
@@ -231,18 +231,14 @@ describe('Pre payment controller', () => {
           }
         }
         const res = {}
-        const next = sinon.spy()
 
-        controller(req, res, next)
-
-        const expectedError = sinon.match.instanceOf(InvalidPrefilledAmountError)
-          .and(sinon.match.has('message', 'Invalid amount: not-valid-amount'))
+        controller(req, res)
 
-        sinon.assert.calledWith(next, expectedError)
+        sinon.assert.calledWith(mockResponse.response, req, res,'prefilled-link-error')
```

Comment:
Just one assertion that needs updatin

Reply:
Good spot Iqbal. Change made as requested.